Automattic/wp-calypso
# Add Professional email setup card checklist to WordPress dashboard

#### Changes proposed in this Pull Request

Add task to complete Professional email setup
![image](https://user-images.githubusercontent.com/5689927/127175102-df265bca-36ec-4e27-8938-0237a713183a.png)


#### Testing instructions

1. Have a new site created from the onboarding steps
2. Go to dashboard
3. Check tasks lists
4. One of the tasks to complete should be "Setup your professional Email"
5. Complete the setup -> The task should be marked as completed.

Related to {1200182182542585-as-1200588428897562}
